### PR TITLE
feat(1314): Memory option

### PR DESCRIPTION
**Pull request by MysticDoll**
## Context

This PR makes `sd-local` enable to set memory limit for build container. 

## Objective

- Add memory limit options
- Refactor passing arguments to launch.New

## References
https://github.com/screwdriver-cd/screwdriver/issues/1314
https://github.com/screwdriver-cd/screwdriver/blob/master/design/sd-local.md#options

## License

I confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

File: cmd/build.go [hunk 1]
```diff
@@ -123,6 +134,13 @@ func newBuildCmd() *cobra.Command {
 		launch.ArtifactsDir,
 		"Path to the host side directory which is mounted into $SD_ARTIFACTS_DIR.")
 
+	buildCmd.Flags().StringVarP(
+		&memory,
+		"memory",
+		"m",
+		"",
+		"Memory limit for build container. Same as the option for docker.")
```
Comment (kkisic) [inline cmd/build.go]: Is it better to add description for format?

Comment (sakka2) [inline cmd/build.go]: Do we need to add the following message to the description which is written in design doc?
`Either b, k, m, g can be used as a size unit`

Comment (yuichi10) [reply cmd/build.go]: It is more helpful, I will add the explain.

Comment (yuichi10) [reply cmd/build.go]: I will add the unit explain.

**Pull request merged**